### PR TITLE
Fixes print statement on python3

### PR DESCRIPTION
**Pull request by misl6**
Fixes print statements for python3.
Now the user defined sdl2 include dir is preferred when declared. (Fixes immintrin.h "not found").

I think that also fixes the issue #23

### CODE REVIEW

File: docs/source/api.rst [hunk 1]
```diff
@@ -52,7 +52,7 @@ Core API
         from audiostream import get_input
 
         def mic_callback(buf):
-            print 'got', len(buf)
+            print ('got', len(buf))
```
Comment (KeyWeeUsr) [inline docs/source/api.rst]: print is a function

File: docs/source/examples.rst [hunk 2]
```diff
@@ -28,7 +28,7 @@ Example to read microphone bytes::
 
     # declare a callback where we'll receive the data
     def callback_mic(data):
-        print 'i got', len(data)
+        print ('i got', len(data))
```
Comment (KeyWeeUsr) [inline docs/source/examples.rst]: print is a function

File: examples/record_mac.py [hunk 3]
```diff
@@ -1,14 +1,18 @@
+from __future__ import print_function
```
Comment (KeyWeeUsr) [inline examples/record_mac.py]: no need for that, just use the brackets correctly :)

File: setup.py [hunk 4]
```diff
@@ -19,23 +20,24 @@
...
         cmdclass['build_ext'] = build_ext
     except ImportError:
-        print '**** Cython is required to compile audiostream ****'
+        print('**** Cython is required to compile audiostream ****')
         raise
 
 # configure the env
 use_sdl2 = environ.get('USE_SDL2')
 if use_sdl2:
-    include_dirs = ['/usr/include/SDL2']
```
Comment (KeyWeeUsr) [inline setup.py]: any particular reason for this?

Comment (misl6) [reply setup.py]: The "immintrin.h" header was requested by "SDL_cpuinfo.h" located in /usr/include/SDL2 .

When compiling via the python-for-android "audiostream" recipe, this causes an error.

I think that at least for the Android release, we have to refer to the SDL2_INCLUDE_DIR env variable.

I'm appending it later as a fallback. (Not sure if is needed).

File: examples/record_mac.py (hunk 3 above)
Comment (misl6) [reply examples/record_mac.py]: For me it's ok to remove.
What about print('i got', len(data)) in python2 ? Without this import returns a tuple.

File: setup.py (hunk 4 above)
Comment (inclement) [reply setup.py]: If it works, this seems okay to me. It makes sense that the env var should have priority.

File: examples/record_mac.py (hunk 3 above)
Comment (misl6) [reply examples/record_mac.py]: You're right. Removed.

File: setup.py (hunk 4 above)
Comment (misl6) [reply setup.py]: @inclement , tested on Android w/ python3 and it worked.

**Pull request merged**